[DOCS] Replaces outdated integration guides with redirects to versioned documentation (#8173)
great-expectations/great_expectations: docs/docusaurus/docs/deployment_patterns/how_to_use_great_expectations_in_aws_glue.md

description: AWS Glue
sidebar_custom_props: { icon: 'img/integrations/aws_glue_icon.svg' }
---

> AWS Glue is a serverless data integration service by Amazon.  It makes it easier to prepare, move, and integrate data from multiple sources.  It has applications for analytics, machine learning, and application development.

Reviewer: ```suggestion
AWS Glue is a serverless data integration service that makes it easier to prepare, move, and integrate data from multiple sources. It includes applications for analytics, machine learning, and application development.
```

:::info 

> The most recent guide for using GX with AWS Glue is for GX version 0.15.50 and can be read in [GX's versioned integration documentation for AWS Glue](/docs/0.15.50/deployment_patterns/how_to_use_great_expectations_in_aws_glue).

Reviewer: ```suggestion
To use Great Expectations (GX) with AWS Glue, see [How to use GX in AWS Glue](/docs/0.15.50/deployment_patterns/how_to_use_great_expectations_in_aws_glue) in version 0.15.50 of the documentation.
```

> Guidance on AWS Glue's usage can be found on [Amazon's AWS Glue site](https://aws.amazon.com/glue/).

Reviewer: ```suggestion
For more information about implementing and using AWS Glue, see the [AWS Glue site](https://aws.amazon.com/glue/).
```
